src/pages/RepoPage/FailedTestsTab/GitHubActions/FrameworkTabs/FrameworkTabs.tsx

```diff
@@ -27,27 +27,22 @@ export function FrameworkTabs() {
 
   return (
     <div>
-      <div className="flex justify-between">
-        <div className="flex gap-1">
-          {Object.keys(FrameworkCopy).map((f) => (
-            <button
-              key={f}
-              className={cs(
-                'px-4 py-2',
-                selectedFramework === f && 'border-b-2 border-ds-gray-octonary'
-              )}
-              onClick={() => setSelectedFramework(f as FrameworkType)}
-            >
-              {f}
-            </button>
-          ))}
-        </div>
+      <div className="flex gap-1">
+        {Object.values(Frameworks).map((framework) => (
+          <button
+            key={framework}
+            className={cn(
+              'px-4 py-2 border-b-2 border-transparent',
+              selectedFramework === framework && 'border-ds-gray-octonary'
+            )}
```

Comment:
The the old `cs` function this would require us to check both states or else `border-transparent` would always be applied, with the newer `cn` function it will automatically remove the `border-transparent` class as the `border-ds-gray-octonary` comes later so it has precedent over `border-transparent`